ore-rs.gemspec

```diff
@@ -16,14 +16,22 @@ Gem::Specification.new do |s|
 
   s.authors  = ["Matt Palmer"]
   s.email    = ["matt@cipherstash.com"]
-  s.homepage = "https://github.com/cipherstash-ruby-ore-rs"
+  s.homepage = "https://cipherstash.com"
 
   s.files = `git ls-files -z`.split("\0").reject { |f| f =~ /^(\.|G|spec|Rakefile)/ }
 
   s.extensions = ["ext/ore_rs/extconf.rb"]
 
   s.required_ruby_version = ">= 2.7.0"
 
+  s.metadata["homepage_uri"] = s.homepage
+  s.metadata["source_code_uri"] = "https://github.com/cipherstash/ruby-ore-rs"
+  s.metadata["changelog_uri"] = "https://github.com/cipherstash/ruby-ore-rs/releases"
+  s.metadata["bug_tracker_uri"] = "https://github.com/cipherstash/ruby-ore-rs/issues"
+  s.metadata["documentation_uri"] = "https://rubydoc.info/gems/ore-rs"
+  s.metadata["mailing_list_uri"] = "https://discuss.cipherstash.com"
+  s.metadata["funding_uri"] = "https://cipherstash.com/signup"
```

Comment:
Is this deliberate? It doesn't seem like the intended purpose of a funding URI

Reply:
From the convo offline: I'll update this to the pricing page.